docs/user_stories/topology.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Second basic tutorial - topology management
+layout: default
+---
+
+
+This example aims to create a substation with a node/breaker topology model and to visit it through the bus/breaker and the bus views. In a second time, we create the same substation with a bus/breaker topology model and we visit it through the bus/breaker and the bus views.
+
+# Building a network in node/breaker topology model
+
+The substation of this tutorial has two voltage levels VL1 and VL2 described with a node/breaker topology model. The first voltage level VL1 has 2 busbar sections BBS1 and BBS2, a generator GN, a load LD and a coupler BR3 between busbar section BBS1 and BBS2. The second voltage level VL2 has a single busbar section BBS3, a line LN and is connected to voltage level VL1 through transformer TR. Here is a diagram of the substation:
+
+![Node breaker topology](./images/nodeBreakerTopology.svg){: width="50%" .center-image}
+
+The node/breaker topology model is stored inside the voltage level as a graph where connection nodes are the vertices and switches are the edges. The next diagram shows how to map the substation topology to a graph.
+
+![Node breaker graph](./images/nodeBreakerTopologyGraph.svg){: width="50%" .center-image}
+
+- Each voltage level has its own topology graph. Voltage level VL1 has 8 connection nodes. Generator GN is connected to node 1, load LD to node 5, busbar sections BBS1 and BBS2 to node 3 and 4. 2, 6 and 7 are internal connection nodes.
+- Voltage level VL2 has 3 nodes, line LN is connected to node 1, busbar section BBS3 to node 2.
+- Transformer TR is connected to node 8 of voltage level 400 KV and node 3 of voltage level 225 KV.
+- Plain edges represent closed switches.
+- Dashed edges represent opened switches.
```

Comment:
The dashed edge 5<->6 has a different representation than the 3-7 or 2-4 ones, is it normal?
Then it would be worth to explain why I think: it would help better understand the question of the load later on (the fact that it does not appear in the network when creating it directly with a bus/breaker topology).

Reply:
I have fixed the svg file to have the same kind of edge for 5<->6. But I am not sure to understand what you mean in the second paragraph ?